src/pages/lessons/[slug].tsx

```diff
@@ -389,39 +391,43 @@ const Lesson: FunctionComponent<LessonProps> = ({initialLesson}) => {
                     } w-full h-full top-0 left-0`}
                   >
                     {loaderVisible && <Loader />}
-                    {playerVisible && (
-                      <EggheadPlayer
-                        ref={playerRef}
-                        resource={lesson}
-                        hls_url={data.hls_url}
-                        dash_url={data.dash_url}
-                        playing={playerState.matches('playing')}
-                        width="100%"
-                        height="auto"
-                        pip="true"
-                        controls
-                        onPlay={() => send('PLAY')}
-                        onPause={() => {
-                          send('PAUSE')
-                        }}
-                        onProgress={({...progress}) => {
-                          onProgress(progress).then((lessonView: any) => {
-                            if (lessonView) {
-                              setLessonView(lessonView)
-                            }
-                          })
-                        }}
-                        onEnded={() => {
-                          onEnded().then((lessonView: any) => {
-                            if (lessonView) {
-                              setLessonView(lessonView)
-                            }
-                            send('COMPLETE')
-                          })
-                        }}
-                        subtitlesUrl={get(lesson, 'subtitles_url')}
-                      />
-                    )}
+                    <EggheadPlayer
+                      ref={playerRef}
+                      hidden={!playerVisible}
```

Comment:
using `hidden` instead of removing component in attempt to keep a single player instance

couldn't get autoplay with full screen to work